test/utils/AssertAsync.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const { assert } = require('chai');
+
+/**
+ * This class includes the function to assert promise.
```

Comment:
Not a good description. This class only exists to have that one method? Then we don't even need a class...
How about something like "AssertAsync asserts values or errors returned by promises"?